packages/transaction-controller/src/utils/batch.ts

```diff
@@ -58,6 +60,13 @@ type AddTransactionBatchRequest = {
     options: { transactionId: string },
     callback: (transactionMeta: TransactionMeta) => void,
   ) => void;
+  publishTransaction: (
+    _ethQuery: EthQuery,
+    transactionMeta: TransactionMeta,
+  ) => Promise<Hex>;
+  getPendingTransactionTrackerByChainId: (
```

Comment:
Minor, also here, maybe `getPendingTransactionTracker` is better as long otherwise?

Reply:
Done